hosted/json-schemas/rfq.schema.json

```
@@ -8,12 +8,12 @@
       "type": "string",
       "description": "Offering which Alice would like to get a quote for"
     },
-    "claims": {
+    "claimsHashes": {
       "type": "array",
       "items": {
         "type": "string"
       },
-      "description": "Presentation Submission that fulfills the requirements included in the respective Offering"
+      "description": "Digests of Presentation Submissions that fulfills the requirements included in the respective Offering"
```

Do you think it would be good to be slightly more specific (also in the `paymentDetails` in the lines below)

```suggestion
      "description": "Salted hash of Presentation Submissions that fulfills the requirements included in the respective Offering"
```

---

Oh yeah, we have a dedicated section on Digest's so idk maybe not? What do you think @diehuxx? My concern is some places we have language around "digest" (which is more broad) and other places "salted hash" (which is a particular type of digest) and how that may lead to ambiguity and/or confusion.

---

How do you feel about "salted digest"? It sounds much less appetizing than a salted hash, but may be more specific to what we actually mean. I waffled on this one tbh.